grafana/process-dashboard.json

```diff
@@ -0,0 +1,556 @@
+{
+  "__inputs": [
```

Comment:
I didn't see this section in our other dashboards, so maybe is not needed?

Reply:
Yeah. This dashboard is exported with the "Export for sharing externally" option, while the others don't.